web/modules/custom/collection/config/install/core.entity_form_display.rdf_entity.collection.default.yml

```diff
@@ -23,9 +23,10 @@ dependencies:
     - datetime
     - image
     - link
+    - rdf_entity
     - text
 _core:
-  default_config_hash: 7J5kiNcWwzxFV3dpv_JwbNVd6IlpWCqGBp7Y-FzPWGM
+  default_config_hash: fbC_Go0HP9SCUjTRv94g0KHkTip0Bor8FeOJzkymoBk
```

Comment:
This is a recent change in core, these `default_config_hash` parameters should not be committed since they are environment specific.

This is just as information, it does not affect the functionality of this ticket so it doesn't need to be addressed now. config_devel takes care of this.